### PR TITLE
Story/bods 4505

### PR DESCRIPTION
**Pull request by alvak1**
changes to map txcfileattributes id to service table

### CODE REVIEW

File: transit_odp/timetables/extract.py [hunk 1]
```diff
@@ -44,11 +45,18 @@
 class TransXChangeExtractor:
     """An API equivalent replacement for XmlFileParser."""
 
-    def __init__(self, file_obj: File, start_time):
+    def __init__(self, file_obj: File, start_time, df_txc_files=pd.DataFrame()):
+        pd.set_option("display.max_rows", None)
```
Comment (NitishAroraKPMG) [inline transit_odp/timetables/extract.py:49]: remove it

Comment (NitishAroraKPMG) [inline transit_odp/timetables/extract.py:49]: remove it

File: transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py [hunk 2]
```diff
@@ -81,6 +82,38 @@ def create_stop_point_cache(revision_id):
     return create_naptan_stoppoint_df_from_queryset(stops)
 
 
+def create_txc_file_attributes_df(queryset):
+    df_txc_file_attributes = pd.DataFrame.from_records(queryset.values())
```
Comment (NitishAroraKPMG) [inline transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py:86]: Add documentation

Comment (alvak1) [reply transit_odp/pipelines/pipelines/dataset_etl/utils/dataframes.py:89]: done

File: transit_odp/timetables/extract.py (hunk 1 above)
Comment (alvak1) [reply transit_odp/timetables/extract.py]: done

**Review by pritish-ranjan01 — APPROVED** [review]

**Pull request merged**